views/default/input/date.php

```diff
@@ -48,8 +52,20 @@
 	$vars['value'] = gmdate('Y-m-d', $vars['value']);
 }
 
+$datepicker_options = elgg_extract('datepicker_options', $vars);
```

Comment:
Well, if we manage to deprecate them all before 3.x, we can just get rid of
them without any new APIs. It will be a challenge.
On Dec 18, 2015 12:47 AM, "Steve Clay" <notifications@github.com> wrote:

> In views/default/input/date.php
> <https://github.com/Elgg/Elgg/pull/9224#discussion_r47978137>:
>
> > @@ -48,8 +52,20 @@
> >  	$vars['value'] = gmdate('Y-m-d', $vars['value']);
> >  }
> >
> > +$datepicker_options = elgg_extract('datepicker_options', $vars);
>
> On 2nd thought that can wait. We need a big strategy to deprecate most of
> these instead of one by one
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/9224/files#r47978137>.
>